Week1/homework/js-exercises/3-logNumber.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+// if you used CONST for a variables without value , You will get an error message
+// const numberX; ( SyntaxError: Missing initializer in const declaration )
+// but if you used another keyword like let or var 
+// var numberX; let numberX; prints undefined without erroe message
+
+var numberX;
+
+console.log('Guess what is the value of X ? Number or String');
+console.log(numberX); 
+
+var numberX = 7;
```

Comment:
Ah, I think this will give an error (or it should at least), because you've already defined `numberX. Could you take a look at this again?

Reply:
no, there is no error.
it's work because I used **VAR**

Reply:
i know you used var, but Wilgert said in the class we should not use var, but `let` or `const`